### ISSUE TITLE
NixOS: Build failed

### ISSUE THREAD

**Issue by Joshua265** [horriblename/hyprgrass#163]
Hardware: Surface Pro  6
OS: NixOS + Flakes

Issue: When executing `nixos-rebuild switch`, hyprgrass encounters an error when building. 
I tried to convert back to commit  427690aec574fec75f5b7b800ac4a0b4c8e4b1d5 but got the same error.

Below you can find the log of the build.
Thank you for any help.
[hyprgrass-build.log](https://github.com/user-attachments/files/17491628/hyprgrass-build.log)

**Comment by matt1432** [comment]
This should be fixed by my PR: https://github.com/horriblename/hyprgrass/pull/164

**Comment by Joshua265** [comment]
Yes, using your PR worked, thank you.

For anybody else coming here, using the PR commit in your flake like this:
```nix
    hyprgrass = {
      url = "github:horriblename/hyprgrass/070ca398300bf5b9e1a636ca057882c0312c228d";
      inputs.hyprland.follows = "hyprland"; 
    };
```
or applying the PR as a patch solves the issue till it's merged.

**Comment by horriblename** [comment]
fixed by #164
